rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc

```diff
@@ -72,6 +72,7 @@ endif::[]
 
 [id="nodes-about-autoscaling-nodes-additional-resources"]
 == Additional resources
+* link:https://access.redhat.com/solutions/6821651[Troubleshooting: Autoscaling is not scaling down nodes]
```

Comment:
In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html#nodes-about-autoscaling-nodes-additional-resources